viz_scripts/bin/generate_plots.py

```diff
@@ -24,7 +25,8 @@
 args = parser.parse_args()
 
 # Read and use parameters from the unified config file on the e-mission Github page
-download_url = "https://raw.githubusercontent.com/e-mission/nrel-openpath-deploy-configs/main/configs/" + STUDY_CONFIG + ".nrel-op.json"
+# download_url = "https://raw.githubusercontent.com/e-mission/nrel-openpath-deploy-configs/main/configs/" + STUDY_CONFIG + ".nrel-op.json"
+download_url = "https://raw.githubusercontent.com/e-mission/nrel-openpath-deploy-configs/main/configs/" + DYNAMIC_CONFIG + ".nrel-op.json"
```

Comment:
you are still changing this code. Is the expectation that this will be the production version or that it will be changed back to the commented out value?